schema.sql

```diff
@@ -8,3 +8,63 @@ CREATE TABLE animals (
     neutered BOOLEAN,
     weight_kg DECIMAL
 );
+
+-- Table named owners with the required columns
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    fullname CHAR(200),
```

Comment:
- [ Optional] Since names vary in length it is better to use `varchar` instead to save some storage space.